src/Benchmarks/Data/DapperDb.cs

```diff
@@ -58,6 +58,28 @@ public DapperDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<Ap
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
+            {
+                db.ConnectionString = _connectionString;
+                await db.OpenAsync();
+
+                for (int i = 0; i < count; i++)
```

Comment:
OK, I will review https://www.techempower.com/benchmarks/#section=code and see if I agree.